core/src/main/java/com/scalar/db/storage/dynamo/DynamoTableMetadataManager.java

```diff
@@ -127,19 +153,174 @@ private DataType convertDataType(String columnType) {
 
   @Override
   public void deleteTableMetadata(String namespace, String table) {
-    // TODO To implement
-    throw new UnsupportedOperationException();
+
+    HashMap<String, AttributeValue> keyToGet = new HashMap<>();
```

Comment:
Maybe `keyToDelete`?
```suggestion
    HashMap<String, AttributeValue> keyToDelete = new HashMap<>();
```